src/sql-parser/src/parser.rs

```diff
@@ -5305,6 +5314,7 @@ impl<'a> Parser<'a> {
                     )
                 }
             };
+            println!("escaped");
```

Comment:
I think you may have left this in by accident?

Reply:
Oh yikes I did--ty